kpops/config/__init__.py

```diff
@@ -125,6 +125,10 @@ class KpopsConfig(BaseSettings):
         default=OperationMode.MANAGED,
         description="The operation mode of KPOps (managed, manifest, argo).",
     )
+    kafka_topic_resource_label: str | None = Field(
```

Comment:
if this is specific to Strimzi Kafka only, would it make sense to have a new config section and making it required there?

Reply:
Done!